modules/serverless-kafka-sink-security-config.adoc

```diff
@@ -0,0 +1,91 @@
+// Module is included in the following assemblies:
+//
+// * serverless/admin_guide/serverless-kafka-admin.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-kafka-sink-security-config_{context}"]
+= Configuring security for Kafka sinks
+
+_Transport Layer Security_ (TLS) is used by Apache Kafka clients and servers to encrypt traffic between Knative and Kafka, as well as for authentication. TLS is the only supported method of traffic encryption for Knative Kafka.
+
+_Simple Authentication and Security Layer_ (SASL) is used by Apache Kafka for authentication. If you use SASL authentication on your cluster, users must provide credentials to Knative for communicating with the Kafka cluster, otherwise events cannot be produced or consumed.
```

Comment:
I think you need a semicolon before "otherwise" (independent clause - see IBM SG)

Reply:
Updated across docs, thanks!